packages/core/src/utils/graphFacade.ts

```diff
@@ -42,25 +42,26 @@ export class GraphFacade {
         this.graphite.graph.addNode({
           category: NodeCategory.Source,
           type: FilesystemNodeType.Directory,
-          path: fsPath,
+          path: resourceFile,
         });
         this.fsBackend.readdir(fsPath);
       } else if (stat.isFile()) {
         this.graphite.graph.addNode({
           category: NodeCategory.Source,
           type: FilesystemNodeType.File,
           subtype,
-          path: fsPath,
+          path: resourceFile,
         });
-        this.fsBackend.readFile(fsPath);
+        this.fsBackend.readFile(resourceFile);
       }
-      return this.graphite.scheduler.drain();
+      await this.graphite.scheduler.drain();
```

Comment:
Should we add a void return type here to let folks know that nothing is being returned?

Reply:
Good catch, thanks!